packages/client/lib/execution/level.ts

```diff
@@ -33,7 +31,7 @@ export class LevelDB implements DB {
     try {
       value = await this._leveldb.get(key, ENCODING_OPTS)
     } catch (error: any) {
-      if (isTruthy(error.notFound)) {
+      if (error.notFound !== undefined) {
         // not found, returning null
```

Comment:
Then shouldn't we be explicitly returning `null` here if we want that?  If it were me, I'd remove `null` from the monorepo entirely but we're not there yet.

Reply:
I've simplified that catch/try logic and am now explicitly checking for true. I've also typecasted to Buffer | null instead of just Buffer, which matches the return type of the function. In this particular case `value` is initialized as `null`, and `get()` throws when not found, so value would never be `undefined`.

One side note though is that the typecasting is necessary because `get()` returns `string | Buffer`. Are we confident that `get` won't ever return a string in this particular instance? If not, shouldn't we convert to buffer instead of typecasting?